worker/include/RTC/Transport.hpp

```diff
@@ -297,6 +297,7 @@ namespace RTC
 		RTC::RtpDataCounter sendProbationTransmission;
 		uint16_t transportWideCcSeq{ 0u };
 		uint32_t initialAvailableOutgoingBitrate{ 600000u };
+		uint32_t maxOutgoingBitrate{ 0u };
```

Comment:
cosmetic, define `maxOutgoingBitrate` *after* `maxIncomingBitrate` for consistency with the order of the corresponding method definitions.

Reply:
:+1: